polartools/pressure_calibration.py

```diff
@@ -0,0 +1,242 @@
+'''
```

Comment:
Codacy has a code-style suggestion here: https://app.codacy.com/gh/APS-4ID-POLAR/polartools/pullRequest?prid=6720685

Applying `black pressure_calibration.py` (or `black -l 75 pressure_calibration.py`, my preference) can standardize the code style to satisfy both flake8 and Codacy.

The program `black` is available in the *bluesky_2021_1* conda environment I just installed on lift.

See https://github.com/BCDA-APS/use_bluesky/tree/main/python_installation#setup-custom-environment-for-bluesky to setup this environment on your own Linux workstation.

Reply:
I tested black on a dummy file. It's good, but I really don't like how it splits the arguments of a function with many arguments into separate lines (one line per argument). I looked a little into it, and couldn't figure out if there is a way to disable that.

Reply:
By the way, it seems to be easy to add it as a GitHub workflow: https://black.readthedocs.io/en/stable/github_actions.html